server/src/main/java/org/candlepin/dto/api/v1/StatusDTO.java

```diff
@@ -0,0 +1,368 @@
+    private String rulesVersion;
+
+    @ApiModelProperty(example = "1")
+    private String release;
+
+    private boolean standalone;
+
+    private Date timeUTC;
+
+    private String rulesSource;
+
+    @ApiModelProperty(example = "[ \"cores\", \"ram\", \"instance_multiplier\" ]")
+    private Set<String> capabilities;
+
+
+    /**
+     * Initializes a new StatusDTO instance with null values.
+     */
+    public StatusDTO() {
+        // Intentionally left empty
+    }
+
+    /**
+     * Initializes a new StatusDTO instance which is a shallow copy of the provided
+     * source entity.
+     *
+     * @param source
+     *  The source entity to copy
+     */
+    public StatusDTO(StatusDTO source) {
+        super(source);
+    }
+
+    public StatusDTO setResult(boolean result) {
+        this.result = result;
+        return this;
+    }
+
+    public boolean getResult() {
+        return result;
+    }
+
+    public String getVersion() {
+        return version;
+    }
+
+    public StatusDTO setVersion(String version) {
+        this.version = version;
+        return this;
+    }
+
+    public String getRelease() {
+        return release;
+    }
+
+    public StatusDTO setRelease(String release) {
+        this.release = release;
+        return this;
+    }
+
+    public boolean getStandalone() {
+        return standalone;
+    }
+
+    public StatusDTO setStandalone(boolean standalone) {
+        this.standalone = standalone;
+        return this;
+    }
+
+    public Date getTimeUTC() {
+        return timeUTC;
+    }
+
+    public StatusDTO setTimeUTC(Date timeUTC) {
+        this.timeUTC = timeUTC;
+        return this;
+    }
+
+    public String getRulesVersion() {
+        return rulesVersion;
+    }
+
+    public StatusDTO setRulesVersion(String rulesVersion) {
+        this.rulesVersion = rulesVersion;
+        return this;
+    }
+
+    /**
+     * @return the rulesSource
+     */
+    public String getRulesSource() {
+        return rulesSource;
+    }
+
+    /**
+     * @param rulesSource the rulesSource to set
+     *
+     * @return
+     *  A reference to this DTO
+     */
+    public StatusDTO setRulesSource(String rulesSource) {
+        this.rulesSource = rulesSource;
+        return this;
+    }
+
+    public Set<String> getManagerCapabilities() {
+        return this.capabilities != null ? new SetView(this.capabilities) : null;
+    }
+
+    /**
+     * Sets or clears the available Candlepin/manager capabilities. If the given capabilities are
+     * null, any set capabilities will be cleared.
+     *
+     * @param capabilities
+     *  The capabilities to set, or null to clear any existing capabilities
+     *
+     * @return
+     *  A reference to this DTO
+     */
+    public StatusDTO setManagerCapabilities(Collection<String> capabilities) {
+        if (capabilities != null) {
+            if (this.capabilities == null) {
+                this.capabilities = new HashSet<String>();
```

Comment:
Something else that I missed. I know it's not that important, but we don't have to set the type when creating new objects any more. We can just use the diamond operator `new HashSet<>`.

Reply:
Force of habit. I imagine that'll be a continuing theme for some time. Fixed.